src/Microsoft.Azure.SignalR.Common/ServiceConnections/GracefulShutdownMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.Azure.SignalR
+{
+    internal enum GracefulShutdownMode
+    {
+        Off = 0,
+        On = 1,

```

Comment:
move OnAndMigrate to 1 which should be used more frequently for migratable feature? And simple On is hard to understand it's usage, better naming? Besides, add summary for the public class.

Reply:
Maybe simple OnAndMigrateConnections to MigrateConnection?